integration-tests/src/test/collectcoins.test.ts

```diff
@@ -0,0 +1,64 @@
+import { KeyringPair } from '@polkadot/keyring/types';
+import { AUTHORITY_SURI } from 'creditcoin-js/examples/setup-authority';
+import { createCollectcoinsId } from 'creditcoin-js/extrinsics/request-collectcoins';
+import { createAddressId } from 'creditcoin-js/extrinsics/register-address';
+import { POINT_01_CTC } from '../constants';
+import { Blockchain, creditcoinApi } from 'creditcoin-js';
+import { CreditcoinApi } from 'creditcoin-js/types';
+import { testData } from './common';
+
+describe('CollectCoins', (): void => {
+    let ccApi: CreditcoinApi;
+    let authority: KeyringPair;
+
+    const { keyring } = testData;
+    const evmAddress = '0xffffffffffffffffffffffffffffffffffffffff';
+    const badHash = '0xbad';
+    const blockchain: Blockchain = 'Ethereum';
+    const addressId = createAddressId(blockchain, evmAddress);
+
+    beforeAll(async () => {
+        process.env.NODE_ENV = 'test';
+        ccApi = await creditcoinApi('ws://127.0.0.1:9944');
```

Comment:
Don't hard-code, use `(global as any).CREDITCOIN_API_URL`. Also see #269 for reference.